src/js/components/sharedComponents/header/InfoBanner.jsx

```diff
@@ -20,14 +22,29 @@ export default class InfoBanner extends React.Component {
     }
 
     render() {
-        const title = 'New to USAspending: COVID-19 Spending Data';
-        const content = (
+        let title = 'New to USAspending: COVID-19 Spending Data';
+        let content = (
             <p>
             USAspending now has spending data from federal agencies related to the Coronavirus Aid, Relief, and Economic Security (CARES) Act and other COVID-19 appropriations.
                 <button onClick={this.props.triggerModal}> Learn more</button> about the new data and features, or <Link to="/disaster/covid-19">visit the profile page</Link> to explore and download the data today!
             </p>
         );
 
+        if (isBefore(startOfToday(), new Date(2022, 1, 17))) {
+            title = 'USAspending Ending Support for Internet Explorer';
+            content = (
+                <p>
+                    Support for Internet Explorer will end on Friday, 2/18/2022. Please use one of the following browsers to continue to access USAspending.gov:
+                    <ExternalLink url="https://support.microsoft.com/en-us/microsoft-edge/make-the-switch-from-internet-explorer-to-microsoft-edge-a6f7173e-e84a-36a3-9728-3df20ade9b3c">{' '}Microsoft Edge</ExternalLink>
+                    ,
+                    <ExternalLink url="https://support.google.com/chrome/answer/95417?hl=en&co=GENIE.Platform%3DDesktop">{' '}Google Chrome</ExternalLink>
```

Comment:
This is a link to making Chrome the default browser, although it has a link to installing...

Reply:
In the description in the ticket there are three bullet points and one of them says the ideal links have been provided, so I think that means they've been checked and approved.
As to why no link to Safari, it's probably just because that link would not be useful to anyone not using an Apple device, and most of our users are on Windows.